### PR TITLE
feat(ci): adds multi app preview for heureka and doop within PRs

### PR DESCRIPTION
**Pull request by ArtieReus**
# Summary

This PR introduces multi-app deployments, allowing previews of the current state of the PR on GitHub Pages for heureka and doop apps. 

# Changes Made

- Adds a script file to be reused in the workflow
- Deploy PR Workflow extended to deploy also heureka and doop
- Creates an index.html file to be consumed by docker and workflow

# Related Issues

#398 

- Issue 1: [link to issue]

# Screenshots (if applicable)

# Testing Instructions

1. `npm i`
2. `npm run TASK`

# Checklist

- [x] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

File: apps/doop/docker/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Supernova</title>
```
Comment (hodanoori) [inline apps/doop/docker/index.html:4]: Shouldn't be "Doop"?

File: apps/heureka/docker/index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Supernova</title>
```
Comment (hodanoori) [inline apps/heureka/docker/index.html:4]: Here is "heureka"?

File: apps/doop/docker/index.html (hunk 1 above)
Comment (ArtieReus) [reply apps/doop/docker/index.html:4]: right!!

**Review by hodanoori — DISMISSED** [review]
lgtm

File: apps/heureka/docker/index.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  <body>
+    <div id="root"></div>
+    <script type="module">
+      import {mount} from "./{{entry_file}}"
+      const props = await fetch("./appProps.json").then((r) => r.json())
+      const root = document.getElementById("root")
+      mount(root, {props})
+    </script>
+  </body>
+</html>
```
Comment (barsukov) [inline apps/heureka/docker/index.html:25]: new line.... Please configure your editor on auto save to use prettier.

File: apps/doop/docker/index.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  <body>
+    <div id="root"></div>
+    <script type="module">
+      import {mount} from "./{{entry_file}}"
+      const props = await fetch("./appProps.json").then((r) => r.json())
+      const root = document.getElementById("root")
+      mount(root, {props})
+    </script>
+  </body>
+</html>
```
Comment (barsukov) [inline apps/doop/docker/index.html:25]: new line

**Review by barsukov — CHANGES_REQUESTED** [review]
I have few questions :) Please fix new line and also maybe we need to make lint fail on it

https://stackoverflow.com/questions/50162952/vs-code-end-of-file-new-line-not-persisting#:~:text=The%20correct%20setting%20is%20%22files.insertFinalNewline%22%3A%20false%2C%20setting%20this,line%20to%20be%20added%20when%20saving%20the%20file.

So maybe we need it to add this file extension to prettier
https://prettier.io/docs/en/rationale.html#empty-lines

**Review by edda — APPROVED** [review]

**Review by andypf — APPROVED** [review]

**Review by barsukov — APPROVED** [review]

**Pull request merged**